cmd/server/config.go

```diff
@@ -72,6 +73,11 @@ func (c *config) applyDefaults() {
 }
 
 func (c *config) validate() error {
+	// avalanchego node define lowercase network names.
+	// We silently change config to ensure backward compatibility
+	// with previous deployments.
+	c.NetworkName = strings.ToLower(c.NetworkName)
```

Comment:
I've just tested this. Rosetta asserter compares request network with what's provided to it at https://github.com/ava-labs/avalanche-rosetta/blob/pchain-support/cmd/server/main.go#L163-L166. As a result of this, capitalized network names in the requests still fail.